src/vm/virtualcallstub.cpp

```diff
@@ -4254,3 +4254,10 @@ BOOL VirtualCallStubManagerManager::TraceManager(
     // Forward the call to the appropriate manager.
     return pMgr->TraceManager(thread, trace, pContext, pRetAddr);
 }
+
+#ifdef _TARGET_X86_
+void BackPatchWorkerStaticStub(PCODE returnAddr, TADDR siteAddrForRegisterIndirect)
```

Comment:
Need to use ``#if defined(_TARGET_X86_) && defined(FEATURE_PAL)`` to fix x86/Windows build error.